_data/simple-icons.json

```diff
@@ -3610,6 +3610,11 @@
             "hex": "CC0000",
             "source": "https://github.com/jekyll/brand"
         },
+        {
+            "title": "Jellyfin",
+            "hex": "AA5CC3",
```

Comment:
Looking at the [branding page](https://jellyfin.org/docs/general/contributing/branding.html#theme), the colour `#AA5CC3` is used at the start of the gradient, but `#00A4DC` is used at the end, and is mentioned below as the 'theme accent colour'. I would either recommend using `#00A4DC` or `#5580D0` which is the [midpoint between the two](https://meyerweb.com/eric/tools/color-blend/#AA5CC3:00A4DC:1:hex).